portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/logging/CompanyLogging.testcase

```diff
@@ -22,75 +24,398 @@ definition {
+					locator1 = "CompanyLogs#COMPANY_LOG_VIEW");
 
+				SelectWindow.selectPopUp(locator1 = "null");
+
+				AssertElementPresent(
+					key_logContent = "Initializing com.liferay.site.initializer.welcome for group ${groupId}",
+					locator1 = "CompanyLogs#COMPANY_LOG_CONTENT");
+
+				Close.closeWindow();
+			}
+		}
 	}
 
 	@description = "A company admin should only be able to see the logs from the company they are an admin of"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test GetCompanyAdminLogs {
+		task ("Given a new instance is created") {
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "www.able.com",
+				portalInstanceId = "www.able.com",
+				virtualHost = "www.able.com");
+		}
+
+		task ("And given the admin for that virtual instance logs in") {
+			User.logoutPG();
 
-		// TODO LRQA-75855
+			User.firstLoginPG(
+				password = "test",
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
+		}
 
+		task ("When the instance admin navigates to the company logging UI access point") {
+			Navigator.openSpecificURL(url = "http://www.able.com:8080/o/company-log");
+		}
+
+		task ("Then only logs for that instance are shown") {
+			AssertElementPresent(
+				key_virtualHost = "www.able.com",
+				locator1 = "CompanyLogs#COMPANY_LOG_TITLE");
+
+			AssertElementNotPresent(
+				key_virtualHost = "liferay.com",
+				locator1 = "CompanyLogs#COMPANY_LOG_TITLE");
+		}
 	}
 
 	@description = "A company admin that tries to directly download the logs of a company they do not have access to should get a permissions error"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test GetCompanyAdminLogsFromDifferentCompany {
+		task ("Given a new instance is created") {
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "www.able.com",
+				portalInstanceId = "www.able.com",
+				virtualHost = "www.able.com");
+		}
+
+		task ("And given the admin for that virtual instance logs in") {
+			PortalInstances.openVirtualInstancesAdmin();
+
+			var companyId = TestUtils.getElementTextContent(element = "//tr[contains(.,'liferay.com')]/td[1]//a");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log");
+
+			var logName = TestUtils.getElementTextContent(element = "//h1[text()='liferay.com']//following::a[1]");
+
+			var logSize = StringUtil.extractLast("${logName}", "log");
+
+			var logName = StringUtil.replace("${logName}", "${logSize}", "");
+
+			User.logoutPG();
+
+			User.firstLoginPG(
+				password = "test",
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
+		}
+
+		task ("When the company admin tries accessing the company logging UI to download a specific company log from a different company") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log/${companyId}/${logName}?download=true");
+		}
+
+		task ("Then a permissions error is shown and no log is downloaded") {
+			AssertTextPresent(
```

Comment:
This shouldn't be happening since you are logging in with the omni-admin.(They have access to every company). This could either be a bug, or the url we are navigating to here is incorrect if this is passing